### PR TITLE
Bring back script integration

### PR DESCRIPTION
**Pull request by samuel-williams-da**
After some investigation, a new sdk version was needed to correctly import daml-script in snapshots.
This PR brings back #16331 with this behaviour fixed

Run `DAML_SDK_RELEASE_VERSION=2.6.0-snapshot.20230126.11324.0.11111111 bazel test //compiler/damlc/tests:integration-v1dev` locally to verify

### CODE REVIEW

**Review by dylant-da — APPROVED** [review]

**Comment by dylant-da** [comment]
Looks good, thank you for figuring this out! :)

**Review by akrmn — APPROVED** [review]
thank you @samuel-williams-da!

**Comment by samuel-williams-da** [comment]
I'm gonna wait until after the weekly release to merge this, so we have a chance to see the daily run

**Pull request merged**
